bigchaindb/web/views/transactions.py

```diff
@@ -24,6 +26,8 @@
 from bigchaindb.models import Transaction
 from bigchaindb.web.views.base import make_error
 
+logger = logging.getLogger(__name__)
```

Comment:
There doesn't appear to be any use of this within this file, are there more changes to come?